server/data_common/dataset_metadata.py

```diff
@@ -71,7 +71,16 @@ def get_dataset_metadata_for_explorer_location(dataset_explorer_location: str, a
         dataset_metadata = request_dataset_metadata_from_data_portal(
             data_portal_api_base=app_config.server_config.data_locator__api_base, explorer_url=explorer_url_path
         )
+
         if dataset_metadata:
+            if dataset_metadata['tombstoned']:
+                dataset_id = dataset_metadata['dataset_id']
+                collection_id = dataset_metadata['collection_id']
+                msg = f"Dataset {dataset_id} from collection {collection_id} has been tombstoned and is no " \
```

Comment:
I believe for strings the default way to do this is with triple quotes.

Reply:
I think it's cleaner here without using triple quotes, since that requires dedent-ting and removal of return line